### ISSUE TITLE
Elm-test incorrectly runs inside preference window

### ISSUE THREAD

**Issue by austinshenk** [klazuka/intellij-elm#518]
Elm version: 0.19.0-bugfix6
Elm-test version: 0.19.0-rev6
Both are installed at the project level and not globally. I tried running the tests with the Auto-Discover path as well and it worked fine. I also checked if I could run the following and I got the correct version

```
./node_modules/.bin/elm-test --version
```

Preference window with failing error:
![image](https://user-images.githubusercontent.com/2991598/65834011-b32b8b00-e2a4-11e9-829b-882717928565.png)

Test runs still running properly with run configuration:
![image](https://user-images.githubusercontent.com/2991598/65834020-cd656900-e2a4-11e9-91b6-1b82969786ab.png)

**Comment by klazuka** [comment]
Have you tried using an absolute path instead of a relative path?

**Comment by austinshenk** [comment]
Yes, an absolute path is what I got when I used the `Auto Discover` feature. That still threw an error. I forgot to mention that in the initial description.

**Comment by klazuka** [comment]
are you sure that the directory name should not be `.bin` instead of `bin`?
e.g. I use an absolute path that ends in `node_modules/.bin/elm-test` (macOS)
